packages/relative-values/models/src/templates/relative-values/make_ts_file.sh

```diff
@@ -0,0 +1,10 @@
+dir1="/home/loki/Documents/core/software/fresh/js/squiggle/squiggle-animal-model/packages/relative-values/models/src"
+dir2="$dir1/templates/relative-values"
+
+template_file="$dir2/relative-values.template"
+squiggle_code_file="/home/loki/Documents/core/ea/fresh/QURI/fresh/relative-values/attempt_6_animals/simple-reference-animals.squiggle"
+output_file="$dir1/relative-values.ts"
+
+
+sed '/^SQUIGGLE_CODE/ r '$squiggle_code_file $template_file | sed 's/SQUIGGLE_CODE//g' > $output_file
+
```

Comment:
Does this have to be committed? It's pretty specific to your workflow, maybe you could gitignore it or store it outside the repo?

Reply:
Happy to gitignore it. That said, I imagine that having templates is generally a good idea in this case; otherwise iteration becomes really annoying.

Reply:
Done now